test/cli/test-config.js

```diff
@@ -132,7 +132,7 @@ describe('config', () => {
     })
 
     describe('get/set', () => {
-      it('get a config key value', (done) => {
```

Comment:
will tackle all of the CLI tests after http-api compliance with https://github.com/ipfs/js-ipfs/issues/428